FirebaseAuth/Sources/Swift/AuthProvider/GitHubAuthProvider.swift

```diff
@@ -38,7 +38,7 @@ import Foundation
 }
 
 @available(iOS 13, tvOS 13, macOS 10.15, macCatalyst 13, watchOS 7, *)
-class GitHubAuthCredential: AuthCredential, NSSecureCoding {
+@objc(FIRGitHubAuthCredential) class GitHubAuthCredential: AuthCredential, NSSecureCoding {
```

Comment:
Just double checking, is the internal visibility intended for this class?

Reply:
Yes the provider is public, but not the credential: https://github.com/firebase/firebase-ios-sdk/blob/main/FirebaseAuth/Sources/Public/FirebaseAuth/FIRGitHubAuthProvider.h

Reply:
Can the `@objc` attribute go away then if this credential class is not public/open? My understanding is it won't be used by any ObjC code.

Reply:
It's needed since it needs to decode objects encoded by the ObjC-implemented Firebase 10.

Reply:
Ah, I get it. Thanks for explaining, I didn't connect the dots here. Nice job catching this. I never would have thought about it.

Reply:
Testing with the quickstart you developed made it easy to detect. 😄